lib/instrumentation/index.js

```diff
@@ -70,12 +70,17 @@ function Instrumentation (agent) {
   this._patches = new NamedArray()
 
   for (let mod of MODULES) {
-    this.addPatch(mod, (exports, name, version, enabled) => {
-      // Lazy require so that we don't have to use `require.resolve` which
-      // would fail in combination with Webpack. For more info see:
-      // https://github.com/elastic/apm-agent-nodejs/pull/957
-      const patch = require(`./modules/${mod}`)
-      return patch(exports, name, version, enabled)
+    if (!Array.isArray(mod)) mod = [mod]
+    const pathName = mod[0]
+
+    mod.forEach(mod => {
+      this.addPatch(mod, (exports, name, version, enabled) => {
```

Comment:
Rather than this nested loop here, maybe we should just make `addPatch(...)` also accept arrays of module names and put the loop there? 🤔

Reply:
I'm good either way. I guess it doesn't hurt to make the API a little more flexible to support stuff like this better.

@bgardner87 to support this, just add some logic to the `addPatch` function so it also supports being called with an array of strings. I think it would make sense to also add this support to the `removePatch` function as well. After updating the functions, you also need to update [the API docs](https://github.com/elastic/apm-agent-nodejs/blob/master/docs/agent-api.asciidoc#apmaddpatchname-handler) to explain that the first argument to the function can be an array of strings.